buildman_pb/buildman.pb.go

```diff
@@ -948,6 +956,77 @@ func (x *BuildPack_GitPackage) GetPrivateKey() string {
 	return ""
 }
 
+type SetPhaseRequest_PullMetadata struct {
+	state         protoimpl.MessageState
+	sizeCache     protoimpl.SizeCache
+	unknownFields protoimpl.UnknownFields
+
+	RegistryUrl  string `protobuf:"bytes,1,opt,name=registry_url,json=registryUrl,proto3" json:"registry_url,omitempty"`
+	BaseImage    string `protobuf:"bytes,2,opt,name=base_image,json=baseImage,proto3" json:"base_image,omitempty"`
+	BaseImageTag string `protobuf:"bytes,3,opt,name=base_image_tag,json=baseImageTag,proto3" json:"base_image_tag,omitempty"`
+	PullUsername string `protobuf:"bytes,4,opt,name=pull_username,json=pullUsername,proto3" json:"pull_username,omitempty"`
+}
+
+func (x *SetPhaseRequest_PullMetadata) Reset() {
+	*x = SetPhaseRequest_PullMetadata{}
+	if protoimpl.UnsafeEnabled {
+		mi := &file_buildman_proto_msgTypes[14]
+		ms := protoimpl.X.MessageStateOf(protoimpl.Pointer(x))
+		ms.StoreMessageInfo(mi)
+	}
+}
+
+func (x *SetPhaseRequest_PullMetadata) String() string {
+	return protoimpl.X.MessageStringOf(x)
+}
+
+func (*SetPhaseRequest_PullMetadata) ProtoMessage() {}
+
+func (x *SetPhaseRequest_PullMetadata) ProtoReflect() protoreflect.Message {
+	mi := &file_buildman_proto_msgTypes[14]
+	if protoimpl.UnsafeEnabled && x != nil {
+		ms := protoimpl.X.MessageStateOf(protoimpl.Pointer(x))
+		if ms.LoadMessageInfo() == nil {
+			ms.StoreMessageInfo(mi)
+		}
+		return ms
+	}
+	return mi.MessageOf(x)
+}
+
+// Deprecated: Use SetPhaseRequest_PullMetadata.ProtoReflect.Descriptor instead.
```

Comment:
Any reason the function shouldn't log `Deprecated` when this is called, or be removed all-together?

Reply:
That was generated from the proto file